src/main/java/org/mockito/Mock.java

```diff
@@ -76,4 +76,6 @@
     Class<?>[] extraInterfaces() default {};
 
     boolean serializable() default false;
+
+    boolean lenient() default false;
```

Comment:
This is a more a general point, but we probably should add `mockSettings` here and not have all arguments duplicated in the annotation.

Reply:
Why do you think so? It is very nice and clean to configure the parameters directly?

Reply:
Because we duplicate all values from `mockSettings` into this annotation. That feels brittle and easy-to-forget.

Reply:
That's fair. 

I don't see what's brittle here - this is public API that we don't change. We can forget to add annotation but... so what... our users will remind us and they can always submit a PR.

It's a rather small trade-off for having a good looking API :)